ratis-proto/src/main/proto/Raft.proto

```diff
@@ -470,7 +470,7 @@ message SnapshotManagementRequestProto {
 }
 
 message SnapshotCreateRequestProto {
-
+  uint64 creationGap = 1;
```

Comment:
do we need to set this to optional for compatibility

Reply:
In proto 3, `optional` is generally discouraged; see https://cloud.google.com/apis/design/proto3
> Beginning in protobuf v3.14, primitive fields can distinguish between the default value and unset value by using the [optional keyword](https://cloud.google.com/apis/design/design_patterns.md#optional_primitive_fields), although this is generally discouraged.

In our case, the default value 0 means that it will use server configuration.  So, it is compatible to the previous code.